cmd/katapult/config_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/krystal/katapult-cli/config"
+)
+
+const mockConfigFormat = `---
+api_key: %s
+api_url: %s
+
+`
+
+func TestConfig(t *testing.T) {
+	conf, err := config.New()
+	if err != nil {
+		t.Fatal(err)
+	}
+	tests := []struct {
+		name string
+
+		apiKey string
+		apiURL string
+	}{
+		{
+			name:   "Check that the config command displays blank values properly",
```

Comment:
Nitpick: I think these test case names can be shorted quite a bit as there's lots of repeated words. This one for example I think is enough if it was just `"empty values"`.